foreshadow/transformers/internals/dummyencoder.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import pandas as pd
+from sklearn.base import TransformerMixin, BaseEstimator
+from sklearn.utils import check_array
+from sklearn.utils.validation import check_is_fitted
+
+
+class DummyEncoder(BaseEstimator, TransformerMixin):
+    """Dummy encodes delimmited data within column of dataframe"""
+
+    def __init__(self, delimeter=","):
```

Comment:
We should also handle infrequent categories as part of the requirement. Basically collapse them into an 'other' category.